addons/mail/static/src/models/rtc_call_viewer/rtc_call_viewer.js

```diff
@@ -92,35 +92,9 @@ function factory(dependencies) {
          * @param {boolean} force Force the fullScreen state.
          */
         async toggleFullScreen(force) {
-            if (!this.exists()) {
-                return;
-            }
-            const el = document.body;
-            const fullScreenElement = document.webkitFullscreenElement || document.fullscreenElement;
-            if (force !== undefined ? force : !fullScreenElement) {
-                try {
-                    if (el.requestFullscreen) {
-                        await el.requestFullscreen();
-                    } else if (el.mozRequestFullScreen) {
-                        await el.mozRequestFullScreen();
-                    } else if (el.webkitRequestFullscreen) {
-                        await el.webkitRequestFullscreen();
-                    }
-                    this.update({ isFullScreen: true });
-                } catch (e) {
-                    this.update({ isFullScreen: false });
-                }
-                return;
-            }
-            if (fullScreenElement) {
-                if (document.exitFullscreen) {
-                    await document.exitFullscreen();
-                } else if (document.mozCancelFullScreen) {
-                    await document.mozCancelFullScreen();
-                } else if (document.webkitCancelFullScreen) {
-                    await document.webkitCancelFullScreen();
-                }
-                this.update({ isFullScreen: false });
+            const isFullScreen = await this._toggleFullScreen(force);
```

Comment:
Avoid toggle method, split into enable and disable instead and call explicitly those that we want when we want.